landingPages/initialLaunch/index.html

```diff
@@ -1,14 +1,93 @@
+		</div>
+	</header>
+
+	<section class="grid">
+		<div class="headline">
+			<h1>Your Luxury Limousine and Total Transportation Service.</h1>
+			<h2 class="h2-center">Arriving July 2015.</h2>
+			<p class="headline-copy">The Kevin Smith Transportation Group is the Philadelphia region's newest luxury limousine and total transportation service. We hope to change the way you think about limousine, airport pickup, and towncar service by providing you with a customized experience you'll love.</p>
+		</div>
+		<div class="banner-container">
+			<div class="banner row">
+				<figure class="col-1-4">
+					<img src="images/corporate.png" alt="Corporate" height="230" width="280">
+					<figcaption>Corporate</figcaption>
+				</figure>
+				<figure class="col-1-4">
+					<img src="images/personal.png" alt="Personal and events" height="230" width="280">
+					<figcaption>Personal & Events</figcaption>
+				</figure>
+				<figure class="col-1-4">
+					<img src="images/travel.png" alt="Travel" height="230" width="280">
+					<figcaption>Travel</figcaption>
+				</figure>
+				<figure class="col-1-4">
+					<img src="images/wedding.png" alt="Wedding" height="230" width="280">
+					<figcaption>Wedding</figcaption>
+				</figure>
+			</div>
+		</div>
+	</section>
+
+	<aside class="grid">
+		<div class= "info-container row">
+			<div class="jobs-container col-1-2">
+				<h2>Come work for us</h2>
+				<p>Coming soon...</p>
+<!-- 				<p>We are currently hiring drivers and other positions for our launch this July. Please apply using the job board of your choice.</p>
+				<h3>Job Title Here</h3>
+				<div class="job-listing"><a href="">Apply on Monster &raquo;</a><a href="">Apply on Indeed &raquo;</a><a href="">Apply on Idealist &raquo;</a></div>
+				<h3>Job Title Here</h3>
+				<div class="job-listing"><a href="">Apply on Monster &raquo;</a><a href="">Apply on Indeed &raquo;</a><a href="">Apply on Idealist &raquo;</a></div>
+				<h3>Job Title Here</h3>
+				<div class="job-listing"><a href="">Apply on Monster &raquo;</a><a href="">Apply on Indeed &raquo;</a><a href="">Apply on Idealist &raquo;</a></div> -->
+			</div>
+			<div class="contact-container col-1-2">
+				<div>
+					<h2>Connect with us</h2>
+						<h3>Adam Sitsis</h3>
+						<p>Chief Strategy Officer</p>
+						<div class="contact-info">
+							<p><img src="images/icon-email.svg" alt="email icon" height="40" width="30"><span class="inline-contact"><a href="mailto:asitsis@kevinsmithgroup.com">asitsis@kevinsmithgroup.com</a></span></p>
+							<p><img src="images/icon-phone.svg" alt="phone icon" height="40" width="30"><span class="inline-contact">844.546.674 (844.LIMO.KSG)</span></p>
+					</div>
+				</div>
+				<div>
+					<h3>Follow Us</h3>
+					<div class="social-media">
+						<a href="https://www.facebook.com/kevinsmithgroup"><img src="images/icon-facebook.svg" alt="Facebook" height="29" width="29"></a>
```

Comment:
may want to consider having these open in a new window since they're taking the user away from the page